projects/admin/src/app/circulation/patron/patron-transactions/patron-transaction-event-form/patron-transaction-event-form.component.html

```diff
@@ -0,0 +1,36 @@
+<ng-container *ngIf="transactions">
+  <form [formGroup]="form" (ngSubmit)="onSubmitForm()">
+    <div class="modal-header">
+    <h5 class="modal-title">{{ action }}</h5>
+    <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+      <span aria-hidden="true" data-dismiss="modal" (click)="closeModal()">&times;</span>
+    </button>
+  </div>
+    <div class="modal-body">
+      <div class="card mb-4" *ngFor="let transaction of transactions">
+        <div class="card-body">
+          <h5 class="card-title">{{ transaction.type }}</h5>
+          <div class="card-text">
+            <div class="row" *ngIf="transaction.note">
+              <label class="col-4 label-title" translate>Note</label>
```

Comment:
In the other views, we used col-lg-* or col-md-* for responsiveness. Just check the behaviour on small screen devices.